gulpfile.js

```diff
@@ -246,24 +246,28 @@ gulp.task('default', [
         'ckeditor',
         'angular-chosen-img',
         'translations'
-    ], function () {});
+    ), function () {});
 
 
 /**
  * Extra tasks that have to be called manually. Useful for development.
  */
 
 // Watches changes in JavaScript and templates.
-gulp.task('watch', ['js', 'templates', 'pdf-worker', 'css-site', 'css-projector'], function () {
-    gulp.watch([
+gulp.task('watch', gulp.series(gulp.parallel('js', 'templates', 'pdf-worker', 'css-site', 'css-projector')), function () {
```

Comment:
The `series` can be removed, too.

Does the watcher works for you? Here, it just do the tasks and finishes. I tried to put a `console.log` inside the function, but it won't be called, so the watcher isn't set up on my machine. The version:
```
node_modules/.bin/gulp --version
[08:04:17] CLI version 2.0.1
[08:04:17] Local version 4.0.0
```

Reply:
You are right on the ```series``` since there is only one entry. I just moved along a tutorial for upgrading. But the official documentation says otherwise. Will dig into that.

Will also look at the watcher, I honestly didn't check it since it didn't throw any errors and the result was as expected. Will have a look at how to watch mutliple paths with the new gulp syntax and check that in as well.